server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1097,6 +1098,13 @@ private void loadFile(File file, OperateFilePlan plan) throws QueryProcessExcept
       List<ChunkGroupMetadata> chunkGroupMetadataList = new ArrayList<>();
       try (TsFileSequenceReader reader = new TsFileSequenceReader(file.getAbsolutePath(), false)) {
         reader.selfCheck(schemaMap, chunkGroupMetadataList, false);
+        if (plan.getMetadataCheck() && !loadNewTsFileCheckMetadata(reader)) {
+          throw new QueryProcessException(
+              "loading tsfile metadata is inconsistent with existing timeseries.");
+        }
+      } catch (IOException e) {
+        logger.error("can not get timeseries metadata from {}.", file.getAbsoluteFile());
```

Comment:
change to warn
Only system crash is error

Reply:
ok, already change it to warn